http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/LeoRoutesSprayJsonCodec.scala

```diff
@@ -109,6 +109,7 @@ object LeoRoutesSprayJsonCodec extends DefaultJsonProtocol {
       "defaultClientId" -> obj.defaultClientId.toJson,
       "stopAfterCreation" -> obj.stopAfterCreation.toJson,
       "welderEnabled" -> obj.welderEnabled.toJson,
+      "isUpdating" -> obj.isUpdating.toJson,
```

Comment:
Should this be in `GetRuntimeFormat` as well?

Reply:
The original ticket was only for listClusters so I didn't add the field to the getCluster response - I didn't think it was necessary in getCluster since the UI bases the Runtime Manager off of the listCluster calls, but I can add it to getCluster if that would be helpful for AoU/otherwise

Reply:
My intuition is that it should be in getCluster even if not used.. I think of listCluster response as a subset of getCluster.